scripts/update/01-run.sh

```diff
@@ -27,16 +27,6 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
     echo "============================================="
     echo
 
-    # In Umbrel OS v0.1.2, we need to bind Avahi to only
```

Comment:
Are you able to undo this deletion?

We normally keep one PR down to a single atomic change and squash it down into a single commit when we merge to keep the commit history clean.